OpenRA.Mods.Common/Activities/HarvesterDockSequence.cs

```diff
@@ -98,18 +106,37 @@ public override bool Tick(Actor self)
 			throw new InvalidOperationException("Invalid harvester dock state");
 		}
 
+		public override void Cancel(Actor self, bool keepQueue = false)
+		{
+			notifyDockables.Do(d => d.Canceled());
```

Comment:
Better to use foreach(var d in notifyDockables) here and in the other places for consistency.

Reply:
sure